casa/OS/HostInfo.h

```diff
@@ -131,12 +131,22 @@ class HostInfo
     static ptrdiff_t swapFree();
     // </group>
 
+    // Introduce run-time memory limit re-specification capabilities
+    // <group>
+    static void setMemoryTotal(int memory);
```

Comment:
technically yes but you cannot set them back to their original zero value as you will set them to the aipsrc value instead. this is probably fine in practice as the aipsrc should not change during the execution. But returning the old value in the setter is still more convenient